// @mercadopago/sdk-react/cjs/coreMethods/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.updateCardToken = exports.createCardToken = exports.getIssuers = exports.getInstallments = exports.getPaymentMethods = exports.getIdentificationTypes = void 0;
const getIdentificationTypes_1 = __importDefault(require("./getIdentificationTypes"));
exports.getIdentificationTypes = getIdentificationTypes_1.default;
const getPaymentMethods_1 = __importDefault(require("./getPaymentMethods"));
exports.getPaymentMethods = getPaymentMethods_1.default;
const getInstallments_1 = __importDefault(require("./getInstallments"));
exports.getInstallments = getInstallments_1.default;
const getIssuers_1 = __importDefault(require("./getIssuers"));
exports.getIssuers = getIssuers_1.default;
const create_1 = __importDefault(require("./cardToken/create"));
exports.createCardToken = create_1.default;
const update_1 = __importDefault(require("./cardToken/update"));
exports.updateCardToken = update_1.default;
